### PR TITLE
Update What's new in ReadMe for v1.0.0 release.

### PR DESCRIPTION
**Pull request by RongGu**
### Ⅰ. Describe what this PR does
Update What's new in ReadMe for v1.0.0 release.

### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: README-zh_CN.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 |![更新](static/bell-outline-badge.svg) 最新进展：|
 |------------------|
+|**最新发布**：2023年4月17日. Fluid v1.0.0  发布! 相关功能等版本更新内容请详情参见 [CHANGELOG](CHANGELOG.md)。|
```
Comment (cheyang) [inline README-zh_CN.md:17]: I think it's 2024.

**Review by cheyang — CHANGES_REQUESTED** [review]

File: README-zh_CN.md (hunk 1 above)
Comment (RongGu) [reply README-zh_CN.md:17]: > I think it's 2024.

Thanks for the reviewing. I will fix it.

Comment (RongGu) [reply README-zh_CN.md:17]: > And how about only keeping the first and latest release info?

OK, that's make it short but keep all the important release news currently.

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**